test/unit/std/concept/core_language_test.cpp

```diff
@@ -9,7 +9,7 @@
 
 #include <random>
 
-#include <seqan3/core/concept/core_language.hpp>
+#include <seqan3/utility/detail/exposition_only_concept.hpp>
 #include <seqan3/std/iterator>
 
 #include "auxiliary.hpp"
```

Comment:
Could you also move and rename this test?
test/unit/std/concept/core_language_test.cpp  ->  test/unit/utility/detail/exposition_only_concept_test.cpp 
And double check if there are snipptes or performance tests to move?